script/hatchet.ini

```diff
@@ -1,73 +1,77 @@
 [run]
 # What individual steps of HATCHet should we run in the pipeline?
 # Valid values are True or False
-download_panel=True
-count_reads=True
-genotype_snps=True
-phase_snps=True
-fixed_width=False # True uses older fixed-width versions of some commands
-count_alleles=True
-combine_counts=True
-cluster_bins=True
-loc_clust=True # True uses new locality-aware clustering
-plot_bins=True
-compute_cn=True
-plot_cn=True
+download_panel = True
+count_reads = True
+genotype_snps = True
+phase_snps = True
+fixed_width = False  # True uses older fixed-width versions of some commands
+count_alleles = True
+combine_counts = True
+cluster_bins = True
+loc_clust = True  # True uses new locality-aware clustering
+plot_bins = True
+compute_cn = True
+plot_cn = True
+
+# What chromosome(s) do we wish to process in the pipeline? Leave unspecified to process
+# all chromosomes found in the normal/tumor bam files
+chromosomes =
 
 # Path to reference genome
 # Make sure you have also generated the reference dictionary as /path/to/reference.dict
-reference = "/path/to/reference.fa"
+reference = /path/to/reference.fa
 
 # Make sure you have generated the .bam.bai files at the same locations as these bam files
-normal = "/path/to/normal.bam"
+normal = /path/to/normal.bam
 
 # Space-delimited list of tumor BAM locations
-bams = "/path/to/tumor1.bam /path/to/tumor2.bam"
+bams = /path/to/tumor1.bam /path/to/tumor2.bam
 
 # Space-delimited list of tumor names
-samples = "tumor1 tumor2"
+samples = tumor1 tumor2
 
 # Output path of the run script
-output = "output/"
+output = output/
 
 # How many cores to use for the end-end pipeline?
 # This parameter, if specified, will override corresponding 'processes' parameters in individual <step> sections below.
 processes = 6
 
 [download_panel]
-ref_panel = "1000GP_Phase3"
-refpaneldir = "/path/to/reference/panel"
+ref_panel = 1000GP_Phase3
+refpaneldir = /path/to/reference/panel
 
 [genotype_snps]
 # Reference version used to select list of known germline SNPs;
 # Possible values are "hg19" or "hg38", or leave blank "" if you wish for all positions to be genotyped by bcftools
-reference_version = "hg19"
+reference_version = hg19
 # Does your reference name chromosomes with "chr" prefix?; True or False
-chr_notation = False
+chr_notation = True
```

Comment:
I made `chr_notation` True by default so that there is one less thing for the user to change if they wish to run the HATCHet demo. If `False` is the more common case here, we can change it back.